spec/models/reference_spec.rb

```diff
@@ -17,6 +17,10 @@
 
     it { is_expected.to validate_presence_of :relationship }
     it { is_expected.to validate_length_of(:relationship).is_at_most(500) }
+
+    it 'is invalid when the email address format is incorrect' do
```

Comment:
I'm assuming that other tests would fail if the email validation was too strict?

Reply:
Removed